Opio-Emmanuel-Omona/code-labs
# #155738456 Setup android studio

#### What does this PR do?
This PR sets up Android studio with 
- [x] The correct conventional folder structure
- [x] The correct naming conventions for emulators
- [x] Static Code Analysis Plugins setup

#### Description of Task to be completed?
- [x] Download and install Android Studio
- [x] Start a new Project called Code Labs
- [x] Add Findbugs static code analysis plugin
- [x] Add Pmd static code analysis plugin
- [x] Add checkstyle static code analysis plugin
- [x] Add sonarlint static code analysis plugin
- [x] Update README file

#### How can this be manually tested?
* clone the repo.
``` 
git clone https://github.com/Opio-Emmanuel-Omona/code-labs.git
 ```
* check into branch
```
git checkout ch-setupIDE-155738456
```
* run the gradle files.
```
./gradlew pmd
./gradlew checkstyle
./gradlew findbugs
```
#### What are the relevant pivotal tracker stories?
 [#155738456](https://www.pivotaltracker.com/story/show/155738456)
